src/IECore/StreamIndexedIO.cpp

```diff
@@ -1184,31 +1436,43 @@ void StreamIndexedIO::Index::openStream()
 		m_hasChanged = false;
 
 		f.seekg( 0, std::ios::end );
-		Imf::Int64 end = f.tellg();
+		Imf::Int64 fileLen;
+		Imf::Int64 end = fileLen = f.tellg();
 		f.seekg( end-1*sizeof(Imf::Int64), std::ios::beg );
 
 		Imf::Int64 magicNumber = 0;
-		readLittleEndian( f,magicNumber );
+		readLittleEndian( f, magicNumber );
 
 		if ( magicNumber == g_versionedMagicNumber )
 		{
 			end -= 3*sizeof(Imf::Int64);
 			f.seekg( end, std::ios::beg );
-			readLittleEndian( f,m_offset );
-			readLittleEndian( f,m_version );
+			readLittleEndian( f, m_offset );
```

Comment:
In the future, let's try to keep formatting changes separate from logic changes for commits as big as this.

Reply:
Apologies. I never seem to be able to rebase in a way the reorders commits in a logical order, with appropriate changes in each commit.